docs/Rootless.md

```diff
@@ -33,19 +35,32 @@ endif
-    - Currently, it's only possible to build for the new ABI on macOS
-        - If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
+    - Currently, it's only possible to build for the new ABI on macOS (as only the Xcode tools can build for it)
+        - Unfortunately, new toolchain does not support building for the old ABI. If you want to maintain support for earlier versions, you can grab the toolchain from an earlier Xcode release as specified in [arm64e Deployment](arm64e-Deployment.html) and switch between it and the newer toolchain as desired by setting the `PREFIX` variable to the older toolchain's bin (i.e., `<xcode-ver>.xctoolchain/usr/bin/`) in your project's makefile
     - That being said, developers on other platforms can circumvent this *if necessary*:
-        - By adding `oldabi` as a dependency to their package (preferably only as a last resort)
+        - By [using GitHub Actions to compile your tweaks](https://github.com/p0358/SilentScreenshots/blob/master/.github/workflows/build.yml) and upload them as downloadable artifacts (free for both public and private repos)
+        - By using a macOS virtual machine
+            - On Linux you can use [OSX-KVM](https://github.com/kholia/OSX-KVM) script set
+            - Without working App Store, download full Xcode xip archive and extract it, afterwards move `Xcode.app` to `Applications` folder (you can do this graphically through Finder)
+            - If you want to cut on resources used by the VM:
+                - [Enable SSH access in System Preferences](https://osxdaily.com/2022/07/08/turn-on-ssh-mac/)
+                - Disable the WindowServer daemon with `sudo launchctl disable system/com.apple.WindowServer` and reboot to drop the idle CPU usage and RAM down to ~900 MB (this disables the entire graphical interface of macOS)
+                - Reboot and use Theos over SSH
+        - By using [allemande](https://github.com/p0358/allemande) (static binary converter to old ABI)
+            - This is currently the best solution if you cannot use the Xcode toolchain from macOS
+            - It does not work with tweaks including Swift code (including the Cephei 2.0 library)
+        - By adding `oldabi` as a dependency to their package (preferably only as a last resort or for testing, as this forces download of a system-wide tool that converts the libraries on-the-fly and may cause stability issues for users)
+            - If releasing packages without `oldabi` dependency, make sure to uninstall `oldabi` from your device during testing, to avoid accidentally releasing a tweak that silently relies on it without your knowledge!
```

Comment:
```suggestion
            - If you intend to release packages without the `oldabi` dependency, make sure to uninstall `oldabi` from your device during testing to avoid accidentally releasing a tweak that silently relies on it without your knowledge!
```